Source/Core/DolphinQt/Debugger/CodeViewWidget.cpp

```diff
@@ -182,6 +190,18 @@ void CodeViewWidget::FontBasedSizing()
   verticalHeader()->setMaximumSectionSize(rowh);
   horizontalHeader()->setMinimumSectionSize(rowh + 5);
   setColumnWidth(CODE_VIEW_COLUMN_BREAKPOINT, rowh + 5);
+  setColumnWidth(CODE_VIEW_COLUMN_ADDRESS, fm.width(QStringLiteral("80000000")));
+
+  // this is the longest diassembled instruction i could find, so use it as a reference for how long
```

Comment:
```suggestion
  // This is the longest disassembled instruction I could find, so use it as a reference for how long
```
I think you should probably avoid "I" in comments though as it's not clear who wrote them (without having to git blame...)

Reply:
Mhm, yeah, that's fair. I think I'll try to figure out what the actual longest instruction is and then rephrase that to something better.

Reply:
Alright so I figured out the longest instruction and parameter set, but it turns out that using those values makes the columns noticeably too large for the typical case. I've rephrased the comment though.